source/extensions/filters/common/ext_authz/ext_authz.h

```diff
@@ -62,6 +62,10 @@ struct Response {
   std::string body;
   // Optional http status used only on denied response.
   Http::Code status_code{};
+
+  // A set of metadata returned by the authorization server, that will be emitted as filter's
+  // dynamic metadata that other filters can leverage.
+  ProtobufWkt::Struct dynamic_metadata{};
```

Comment:
nit: no need of `{}`

Reply:
Yeah. I was lazy to explicitly initialize some `Response` objects in `source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc`.

Reply:
Updated.